clients/src/main/java/org/apache/kafka/clients/admin/ListConsumerGroupOffsetsOptions.java

```diff
@@ -44,10 +45,21 @@ public ListConsumerGroupOffsetsOptions topicPartitions(List<TopicPartition> topi
         return this;
     }
 
+    /**
+     * Sets an optional requireStable flag.
+     */
+    public void requireStable(final boolean requireStable) {
```

Comment:
Do we need a KIP for this change?

Reply:
Yes :) it's described as the third point in the comment above.

Reply:
Missed it :). Thanks!